packages/cli/src/ceramic-daemon.ts

```diff
@@ -523,26 +543,110 @@ export class CeramicDaemon {
     }
   }
 
-  async startIndexingModels(req: Request, res: Response): Promise<void> {
-    const { modelIDStrings, error } = this._validateModelIDStrings(req.body.models)
-    if (error) {
-      res.status(422).json({ error: error })
+  private async _parseDidJWSAuthHeader(
+    authHeader: string | undefined
+  ): Promise<AdminAPIAuthHeaderContents> {
+    const jwsString = authHeader.split("Authorization: Basic ")[1]
+    const result = await this.ceramic.did.verifyJWS(jwsString)
+    return {
+      kid: result.kid,
+      code: result.payload.code,
+      requestPath: result.payload.requestPath,
+      forModels: result.payload.requestBody ? result.payload.requestBody.models : undefined,
```

Comment:
```suggestion
      forModels: result.payload.requestBody?.models,
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining